statsmodels/genmod/generalized_linear_model.py

```diff
@@ -208,17 +227,21 @@ def __init__(self, endog, exog, family=None, offset=None, exposure=None,
             exposure = np.log(exposure)
         if offset is not None:  # this should probably be done upstream
             offset = np.asarray(offset)
+
+        self.freq_weights = freq_weights
```

Comment:
given that it works we don't have to do anything.
But it would be useful if it's possible to refactor either now or at some point in the future to make the code more *obvious*.
I don't know yet.